Index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>My first Webpage</title>
+</head>
+<body>
+    <h1>Jaipur Heritage</h1>
+    <ol>
+        <h2>Albert Hall Museum</h2>
+        <p>The Albert Hall Museum in Jaipur is the oldest museum of the state and functions as the state museum of Rajasthan, India. The building is situated in Ram Niwas garden outside the city wall opposite New gate and is a fine example of Indo-Saracenic architecture. It is also called the Government Central Museum. It was considered one of the best 19th century museums for the variety of its collections. It was renovated in 2008 and reopened as one of the most advanced museums in India.</p>
+        <img src="./Images/Albert_Hall_Museum.jpg" alt="jaipur">
```

Comment:
Pls make all img tags self-closing. If the tag doesn't have any content, we use the self-closing syntax